invenio_rdm_records/services/schemas/community_records.py

```diff
@@ -8,14 +8,17 @@
 """Record communities schema."""
 
 from marshmallow import Schema, fields, post_load, validate
+from marshmallow_utils.fields import StrippedHTML
 
 from invenio_rdm_records.services.errors import MaxNumberCommunitiesExceeded
 
 
 class CommunitySchema(Schema):
     """Schema to define a community id."""
 
-    id = fields.String()
+    id = fields.String(required=True)
+    comment = StrippedHTML()
+    require_review = fields.Boolean()
```

Comment:
shouldn't it be load only fields?

Reply:
Yes, will fix it! thanks!

Reply:
see my comment above about the comment field